packages/components/src/components/tooltip/tooltip.stories.ts

```diff
@@ -0,0 +1,243 @@
+    })
+};
+
+export const Disabled = {
+  parameters: { controls: { exclude: 'disabled' } },
+  render: (args: any) =>
+    generateTemplate({
+      axis: {
+        x: {
+          type: 'attribute',
+          name: 'disabled'
+        }
+      },
+      args,
+      constants: [{ type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' }]
+    })
+};
+
+/**
+ * Use the `default` and `content` slots in case you need to change them. Additionally a headline can be shown by bolding the text. Links or buttons (interactive elements) are not allowed.
+ *
+ */
+
+export const DifferentSlots = {
+  parameters: {
+    controls: { exclude: ['default', 'content'] }
+  },
+  render: (args: any) => {
+    return html` ${['default', 'content'].map(slot =>
+      generateTemplate({
+        axis: {
+          x: {
+            type: 'slot',
+            name: slot,
+            title: 'slot=...',
+            values: [
+              {
+                value:
+                  slot === 'default'
+                    ? `<span>Tooltip</span>`
+                    : `<div slot='content'><b>Headline</b><br>Lorem ipsum</div>`,
+                title: `${slot} slot`
+              }
+            ]
+          }
+        },
+        args
+      })
+    )}`;
+  }
+};
+
+/**
+ * In case of having the content longer than the space available, the tooltip can be set to scrollable by overriding the `pointer-events` property to auto. This behavior is heavily discouraged, but it is available in case of need.
+ *
+ */
+export const LongContent = {
+  parameters: {
+    controls: { exclude: 'content' }
+  },
+  render: () => {
+    return generateTemplate({
+      axis: {
+        x: {
+          type: 'slot',
+          name: 'content',
+          values: [
+            {
+              value: `<div slot='content' style='pointer-events:auto; width:100px;'>Lorem ipsum sic semper dolor sit amet, consectetur adipiscing elit. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliquet nunc, quis aliquam nisl nisl quis nisl. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliquet nunc, quis aliquam nisl nisl quis nisl. Lorem ipsum sic semper dolor sit amet, consectetur adipiscing elit. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliquet nunc, quis aliquam nisl nisl quis nisl. Nulla euismod, nisl quis ultrices aliquam, nunc nisl aliquet nunc, quis aliquam nisl nisl quis nisl.</div>`,
+              title: 'long with fixed width'
+            }
+          ]
+        }
+      },
+      args: overrideArgs([
+        {
+          type: 'attribute',
+          name: 'open',
+          value: true
+        },
+        {
+          type: 'attribute',
+          name: 'placement',
+          value: 'bottom'
+        }
+      ]),
+      constants: [
+        {
+          type: 'template',
+          name: 'width',
+          value: '<div style="width: 300px">%TEMPLATE%</div>'
+        }
+      ]
+    });
+  }
+};
+
+export const Slots = {
```

Comment:
why do we have 2 slot stories here? This one and the one in l122... It seems to be doubling the information a bit.

Reply:
Yes, I already removed the other one! :)